client/components/Gantt.ts

```diff
@@ -159,7 +159,7 @@ export class Gantt<R, T> {
   ) => void;
   public onEndResize?: (resizedBar: GanttBar<T>, bars: GanttBar<T>[]) => void;
   public readonly tooltipNode: HTMLElement | null;
-
+  public onPan?: (timeRangeStart: Date, timeRangeEnd: Date) => void;

```

Comment:
Replace with
```ts
  public pan?: {
    mouseModifier: [...MouseModifier[], MouseModifier];
    onPan: (timeRangeStart: Date, timeRangeEnd: Date) => void;
  };
```

with
```ts
type MouseModifier = Pick<
  MouseEvent,
  "ctrlKey" | "metaKey" | "altKey" | "shiftKey"
>;
```

check that all the modifiers are enabled so that the user can specify which modifier(s) keys to use.